.buildkite/pipeline.yml

```diff
@@ -3,7 +3,7 @@ steps:
     key: unit_tests
     agents:
       queue: v3
-    command: yarn test
+    command: bash test.sh
```

Comment:
Shouldn't need to specify `bash` here

Reply:
Yes you do since alpine is super lame and uses ash by default

Reply:
But you are specifying the intepreter in the script?

Reply:
I'm fairly certain ./test/sh didn't work

Reply:
Try it again, I really don't see why this is necessary